e2etest/bgptests/infra_setup.go

```diff
@@ -177,26 +228,26 @@ func InfraTearDown(cs *clientset.Clientset, containers []*frrcontainer.FRR) erro
 
 // multiHopSetUp connects the ebgp-single-hop container to the multi-hop-net network,
```

Comment:
can you change this comment + update the docs (can be in a different pr I guess)?